ci-scripts/yocto.groovy

```diff
@@ -87,6 +95,25 @@ void buildImageAndSDK(String yoctoDir, String imageName, boolean dev=true) {
     }
 }
 
+void runSmokeTests(String yoctoDir, String imageName) {
+    String archiveDir = "testReports-" + imageName
+
+    try {
+        stage("Perform smoke testing") {
+            vagrant("/vagrant/cookbook/yocto/runqemu-smoke-test.sh ${yoctoDir} ${imageName}")
+        }
+    } catch(e) {
```

Comment:
I really dislike this way of catching everything and then not even printing what actually happened because this makes impossible to find what actually happened. Either remove the catch-all or make it more specific for some type of a error and print the call stack or dive some possibility to find out what happened instead of just printing a string.

Reply:
See the newly added code below, @rpannek. Is this what you had in mind?